packages/components/organisms/f-header/src/components/CountrySelector.vue

```diff
@@ -0,0 +1,294 @@
+            return this.isBelowMid && !this.countrySelectorIsOpen;
+        },
+        countrySelectorIsOpenOnDesktopView () {
+            return !this.isBelowMid && this.countrySelectorIsOpen;
+        }
+    },
+    methods: {
+        onCountrySelectorToggle () {
+            this.countrySelectorIsOpen = !this.countrySelectorIsOpen;
+        },
+
+        closeCountrySelector () {
+            this.countrySelectorIsOpen = false;
+        },
+
+        openCountrySelector () {
+            this.countrySelectorIsOpen = true;
+        }
+    }
+};
+</script>
+
+<style lang="scss">
+
+$countrySelector-icon-color  : $color-secondary;
+$countrySelector-text-color  : $grey--darkest;
+$countrySelector-text-hover  : $color-bg--darker;
+$countrySelector-flag-width  : 16px;
+$countrySelector-flag-height : 16px;
+
+.c-nav-popoverList-header {
+    display: flex;
+    align-items: center;
+
+    @include media('>=mid') {
+        padding-bottom: spacing(x2);
+    }
+
+    h3 {
+        @include font-size(heading-s, true, narrow);
+        font-weight: $font-weight-headings;
+        margin: 0;
+        margin-left: spacing();
+
+        @include media('>=mid') {
+            @include font-size(heading-s);
+            margin-left: 0;
+        }
+    }
+}
+
+.c-nav-popoverList-header-button {
+    padding: 0;
+    margin: spacing(x2);
+
+    @include media('>=mid') {
+        display: none;
+    }
+}
+
+.c-nav-popoverList-go-back-icon {
+    transform: rotate(180deg);
+    width: 28px;
+    height: 28px;
+    display: inline-block;
+
+    use {
+        fill: $countrySelector-icon-color;
+    }
+}
+
+.c-countrySelector-flag {
+    height: $countrySelector-flag-height;
+    width: $countrySelector-flag-width;
+    margin-right: spacing();
+}
+
+.c-countrySelector-flag--current {
+    margin-right: 0;
+}
+
+.c-countrySelector-currentFlag-wrapper {
+    height: $countrySelector-flag-height;
+    width: $countrySelector-flag-width;
+
+    @include media('<mid') {
+        margin-right: spacing();
+    }
+
+    .c-header--highlightBg &,
+    .c-header--transparent & {
+        @include media('>=mid') {
+            background-color: $white;
+            width: 32px;
+            height: 32px;
+            border-radius: 50%;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+        }
+    }
+}
+
+.c-countrySelector-title {
+    width: 0;
+    overflow: hidden;
+    @include font-size(heading-s, true, narrow);
+
+    @include media('<mid') {
+        width: auto;
+    }
+}
+
+.c-countrySelector-list {
+    margin: 0;
+
+    & > li:before {
+        display: none;
+    }
+}
+
+.c-countrySelector-btn {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.c-nav-list-item .c-countrySelector-btn {
+    border-bottom: none;
+}
+
+.c-countrySelector-country {
+    padding: 0;
+    white-space: nowrap;
+    margin-bottom: 0;
+
+    &:hover {
```

Comment:
Do we need a focus state here too?

Reply:
Default focus styles look fine there I think. I will double check and fix it in next PR if needed